src/storage/hummock_test/src/snapshot_tests.rs

```diff
@@ -334,45 +263,15 @@ async fn test_snapshot_backward_range_scan_inner(enable_sync: bool, enable_commi
 
 #[tokio::test]
 async fn test_snapshot() {
-    test_snapshot_inner(false, false).await;
-}
-
-#[tokio::test]
-async fn test_snapshot_with_sync() {
-    test_snapshot_inner(true, false).await;
-}
-
-#[tokio::test]
-async fn test_snapshot_with_commit() {
-    test_snapshot_inner(true, true).await;
+    test_snapshot_inner().await;
```

Comment:
We lost other test_snapshot_inner  tests?

Reply:
> We lost other test_snapshot_inner tests?

fixed. just resumed tests.